### PR TITLE
VRS: Drop ModelScore feature

### LINKED ISSUE #169: VRS: Drop Feature 'ModelScore'
It's originating from Genmod inheritance pattern computation,
incompatible with ClinVar data.

In genmod, ModelScore is computed using `GQ` field,
```
tor@tor-cg:~/repos/cg/genmod/genmod$ grep -nr GQ
vcf_tools/genotype.py:54:        GQ = kwargs.get('GQ', '0')
vcf_tools/genotype.py:111:            self.genotype_quality = float(GQ)
tor@tor-cg:~/repos/cg/genmod/genmod$ grep -nr genotype_quality
commands/summarize_variants.py:265:    #         if genotypes[individual].genotype_quality < gq_treshold:
annotate_models/model_score.py:34:            if gt_call.genotype_quality > 0:
annotate_models/model_score.py:35:                gq = min(gt_call.genotype_quality, 99)
vcf_tools/genotype.py:26:    - genotype_quality FLOAT
vcf_tools/genotype.py:65:        self.genotype_quality = 0
vcf_tools/genotype.py:111:            self.genotype_quality = float(GQ)
```

The GQ field is set statically by the database adaptor cleanup script,
which adds GQ mock data.

### PR DESCRIPTION
**Pull request by torbjorgen**
Closes #169

- [x] Waiting for slurm job 7840658, to generate norm factors vocabulary
- [x] Waiting for slurm job 8071526, to train
  No improvement compared to baseline, 1.9.0 model.
  Running hyperparameter job now.
- [x] Wait for hprm job 8095938 to complete
  Did not outperform v1.9.0 in terms of val loss, auc and categorical accuracy.
However, iteration 6 seemed to be the most promising;
```
batch_size=128
feature_selection_regularisation=False
embedding-dimensions=6
branch_dense_0=224
branch_dense_1=128
feature_multicollinearity_regularisation=False
dense-activation=linear
dense-layers=5
dense-units=160
dense-units-reduction=0.12000000000000001
dropout_rate=0.4
optimizer=Adam
learning-rate=0.0001
```
 - [x] Run iter 6 for longer
  Add warmup iterations of at least 15 epochs
  Job ID 8113124 (bugged in early stopping)
  Job ID 8125079
- [x] Wait for increased span hpt parameter job to complete: 8215953
- [x] Wait for modified hpt and increased early stopping tol to complete, 8279165
  Run suggests to favor higher end of learning rate (1E-4), embedding dims > 18, 1-2 dense layers, no feature selection regularisation, ~300 dense units, branch_dense_1 <~100, multicollinearity_regularisation penalty <~ 1E-8, dense unit reduction >~0.15
- [x] Waiting to rerun training on commit `11b46c6`
  Improved a bit, but not better than baseline including modelscore feature. Seems like more embedding dimensions is better?
- [x] Wait for hpt run 8318684 (best hpt params)
  Got a model that's potentially stable learning gradient, but learning is slow. Started longrunnig job to test this.
- [ ] Wait for long run training job (no hpt) 8347348

### CODE REVIEW

**Comment by torbjorgen** [comment]
Closed, fixed in separate branch that's merged to master.
